build/testing/cypress/.applitools/2018-12-07-1048_ff6aeb57-9cf2-4611-930b-fdbd7dfc2d96/ff6aeb57-9cf2-4611-930b-fdbd7dfc2d96.html

```diff
@@ -0,0 +1,1439 @@
+<rect fill="black" fill-opacity="0.5" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(6)" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(12)" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(0 6)" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(6 6)" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(12 6)" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(0 12)" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(6 12)" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(12 12)" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(3 3)" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(9 3)" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(3 9)" width="3" height="3"></rect>
+<rect fill="black" fill-opacity="0.5" transform="translate(9 9)" width="3" height="3"></rect>
+</svg></SPAN></BUTTON><SPAN class="c014 c015"><SELECT class="c016"><OPTION value="light">light</OPTION><OPTION value="dark">dark</OPTION><OPTION value="custom">custom</OPTION></SELECT></SPAN><SPAN class="c014"><SELECT class="c016"><OPTION >en</OPTION><OPTION >en-rtl</OPTION></SELECT></SPAN></DIV></DIV></DIV></DIV>
+</BODY></HTML>
```

Comment:
What is this file and why is it being included in a documentation PR?

Reply:
build/test files.  These should be ignored.  Not sure why they were committed.  Will update ignore file in future PR.